frontend/src/fixtures/reviewFixtures.js

```diff
@@ -0,0 +1,58 @@
+const reviewFixtures = {
+  oneReview: {
+    id: 1,
+    reviewerId: 101,
+    itemId: 201,
+    dateServed: "2023-11-01T12:00:00",
+    stars: 3,
+    reviewText: "soggy",
+    status: "Awaiting Moderation",
+    modId: null,
+    modComments: "",
```

Comment:
I would make modComments null if awaiting approval, so we can differentiate between unreviewed vs no comments

Reply:
fixed